### ISSUE TITLE
Unable to insert GIFs into server world without OP

### ISSUE THREAD

**Issue by reiyua** [LOOHP/ImageFrame#81]
Hello!

I am part of a YTSMP of sorts, and me and the host are wondering how to allow non operators to insert GIFS into the world?

**Comment by LOOHP** [comment]
You need to give your players the animated map creation permission. I think it is listed on the SpigotMC resource page permissions section.

**Comment by reiyua** [comment]
I do see the permission section, but am wondering where the actual permissions would be located to implement these permissions. The config.yml file within the server config does not contain any of that...

Cheers :)

**Comment by reiyua** [comment]
There also seeems to be other people having the same issue, being that the default is "op" but what do we change it to to allow everyone?

**Comment by LOOHP** [comment]
Yes, you can allow that permission in the default group of your permission plugin (like LuckPerms)
